lib/drivers/animasDriver.js

```diff
@@ -2085,6 +2085,7 @@ module.exports = function (config) {
             debug('[Hand-off to simulator] Unhandled type!', datum.type);
         }
       }
+      simulator.finalBasal();
```

Comment:
So, thinking out loud here to check my understanding... We didn't have a `finalBasal` method in the simulator before now analogous to the method used for OmniPod & Tandem (looking up settings as of upload and guessing at duration of basal) because you have to suspend the pump in order to upload an Animas, correct? And in the "normal" case (not the case this PR is about), the pump *has* already gone to rate `0` by upload and so that suspend segment would be the final basal, and so we don't have any way to guess about the duration?